components/button/button.tsx

```diff
@@ -157,7 +163,7 @@ export default class Button extends React.Component<ButtonProps, any> {
 
   render() {
     const {
-      type, shape, size, className, htmlType, children, icon, prefixCls, ghost, ...others
+      type, shape, size, className, children, icon, prefixCls, ghost, loading: _loadingProp, ...rest
```

Comment:
What is `_loadingProp` for?

Reply:
The old way is using omit to drop the `loading` from `this.props`, I use spreading syntax to filtered this prop out (avoid importing omit function). The leading underscore means it is unused, coz the loading effect depends on `this.state.loading`.

BTW, may I ask you if I don't mention you in the conversation, will you receive the notification email? I don't sure should I always mention someone in review conversions.

Reply:
Yes, I can receive notification everywhere in ant-design.